src/include/Types/CLTypeParsed.h

```diff
@@ -183,6 +183,8 @@ inline void from_json(const nlohmann::json& j, CLTypeParsedRVA& p,
       case CLTypeEnum::PublicKey:
         p = PublicKey::FromHexString(j.get<std::string>());
         break;
+      default:
```

Comment:
some cerr maybe:
      std::cerr << "Unsupported type in json."<< std::endl;

Reply:
Good avatar 😄